From f5aede2ed48f1cc7266e792b02f509f7b207e2d6 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Fri, 8 Jul 2011 14:26:16 +1000
Subject: Don't override the device name.

The device name comes from the xorg.conf (or the config backend). If you're
capable of setting the device name in the config, you can set it in the
Identifier line as well.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>

diff --git a/src/xf86PM.c b/src/xf86PM.c
index edcf6f8..97e2134 100644
--- src/xf86PM.c
+++ src/xf86PM.c
@@ -492,9 +492,6 @@ PenMountPreInit(InputDriverPtr drv, InputInfoPtr pInfo, int flags)
 		goto SetupProc_fail;
 	}
 
-	/* this results in an xstrdup that must be freed later */
-	pInfo->name = xf86SetStrOption( pInfo->options, "DeviceName", "PenMount");
-
 	return Success;
 
   SetupProc_fail:
-- 
cgit v0.10.2

